python/instrumentation/openinference-instrumentation-bedrock/src/openinference/instrumentation/bedrock/__init__.py

```diff
@@ -74,19 +77,23 @@ def _client_wrapper(
 
         if bound_arguments.arguments.get("service_name") == "bedrock-runtime":
             client = cast(InstrumentedClient, client)
+
             client._unwrapped_invoke_model = client.invoke_model
-            client.invoke_model = _model_invocation_wrapper(tracer)(client)
+            client.invoke_model = _model_invocation_wrapper(tracer)(client)("invoke_model")
+
+            client._unwrapped_converse = client.converse
+            client.converse = _model_invocation_wrapper(tracer)(client)("converse")
```

Comment:
as discussed, let's break the wrapper for converse and invoke model apart so we don't have one wrapper doing two completely different things